main.cpp

```diff
@@ -44,93 +44,93 @@ I will request a change be made to one of your 10 nouns for the sole purpose of
 //  example)
 //  Noun:       arm                     // 1)
 //  action 1:   the arm extends         // 2)
-arm.extend();                           // 3)
+arm.extend();                          // 3)
 //  action 2:   the arm flexes          // 2)
 arm.flex();                             // 3)
 //  action 3:   the arm rotates conter-clockwise
 arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun: dog
+//  action 1: bark
+dog.bark();
+//  action 2: run
+dog.run();
+//  action 3: sit
+dog.sit();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
-//  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
-//  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
-//  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun: leg
+//  action 1: bend
+leg.bend();
+//  action 2: kick
+leg.kick();
+//  action 3: stretch
+leg.stretch();
+//  3) 
+//  Noun: cat
+//  action 1: purr
+cat.purr();
+//  action 2: climb
+cat.climb();
+//  action 3: run
+cat.run();
+//  4) 
+//  Noun: gun
+//  action 1: shoot
+gun.shoot();
+//  action 2: reload
+gun.reload();
+//  action 3: recoil
+gun.recoil();

```

Comment:
As explained in the instructions, here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
**replace** the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.